### DISCUSSION TITLE
Indices folder is consuming 90% of my disk space.

### DISCUSSION THREAD

**Discussion by sayanTlregtech** [wazuh/wazuh#18140] [General]
Hello I had implemented wazuh using docker implementation and after successfully running it for like 1.5 years now the disk space of 256 GB is going to be full. After checking the disk i found out that the indices were consuming more than 188 GB of the disk space. Please help me or else my new project will fail.

**Comment by mdiego92** [comment]
Hello @sayanTlregtech,

Do you have configured a retention policy for the indices?

- Please refer to the following documentation: https://wazuh.com/blog/wazuh-index-management/

If it's urgent, you can delete the indices manually:
From the web interface (Main Menu > Dev Tools):

`GET _cat/indices?v=true&s=index`

Then you can delete the indices with one of the following commands:
**Delete yesterday:**
`DELETE wazuh-alerts-4.x-2023.08.10`

**Delete August with * as wildcard:**
`DELETE wazuh-alerts-4.x-2023.08.*`

Hopefully, this resolves your issue.
Sorry for the late response.

Looking forward to your feedback.
